pkg/skaffold/schema/latest/config.go

```diff
@@ -220,6 +221,13 @@ type HelmFQNConfig struct {
 type HelmConventionConfig struct {
 }
 
+// MortarDeploy contains the configuration needed for deploying with mortar
+type MortarDeploy struct {
+	Name   string `yaml:"name,omitempty"`
+	Source string `yaml:"src,omitempty"`
+	Config string `yaml:"config,omitempty"`
```

Comment:
Are there default values we could apply to `Source` or `Config`?

Reply:
Mortar usage dictates always specifying the source, see: https://github.com/kontena/mortar#deploying-k8s-yaml-manifests

Hence I didn't feel like we should have any defaulting for it in Skaffold either.

By default Mortar looks for a file named `shot.yml` or `shot.yaml` as its config file, see https://github.com/kontena/mortar#shot-configuration-file. As the config file is fully optional to use, I donät think we need to default to anything here.

Reply:
👍